src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js

```diff
@@ -76,38 +77,9 @@ const SubscriberGdprPermissions = ({
             <tbody>
               {state.fields.length ? (
                 <>
-                  {state.fields.map((field, index) => (
-                    <tr key={index}>
-                      <td>{field.name}</td>
-                      <td>
-                        {field.permissionHTML ? (
-                          <S.TextColumn
-                            dangerouslySetInnerHTML={{ __html: field.permissionHTML }}
-                          />
-                        ) : (
-                          <FormattedMessage id="subscriber_gdpr.empty_html_text" />
-                        )}
-                      </td>
-                      <td>
-                        {field.value.toLowerCase() === 'none' ? (
-                          <div className="dp-icon-wrapper">
-                            <span className="ms-icon icon-lock dp-lock-grey"></span>
-                            <FormattedMessage id="subscriber_gdpr.value_none" />
-                          </div>
-                        ) : field.value.toLowerCase() === 'true' ? (
-                          <div className="dp-icon-wrapper">
-                            <span className="ms-icon icon-lock dp-lock-green"></span>
-                            <FormattedMessage id="subscriber_gdpr.value_true" />
-                          </div>
-                        ) : (
-                          <div className="dp-icon-wrapper">
-                            <span className="ms-icon icon-lock dp-lock-red"></span>
-                            <FormattedMessage id="subscriber_gdpr.value_false" />
-                          </div>
-                        )}
-                      </td>
-                    </tr>
-                  ))}
+                  {state.fields.map((field, index) => 
+                    <PermissionExpandableRow field={field} key={index} email={subscriber.email}/>
```

Comment:
Here you are adding the new characteristics for the row, without any flag, and this has a `double service` associated, with no real data, this cannot be into production as it is. 😕

Reply:
You're right, we'll do the feature toggler for this